common/autoware_ad_api_msgs/common/msg/ResponseStatus.msg

```diff
@@ -0,0 +1,14 @@
+# constants for level
+uint16 UNKNOWN=0
+uint16 SUCCESS=1
+uint16 WARNING=2
+uint16 ERROR=3
+
+# constants for code
+uint16 INTERNAL_SERVICE_UNREADY=101
+uint16 INTERNAL_SERVICE_TIMEOUT=102
```

Comment:
I think the prefix `INTERNAL_SERVICE_` can be removed.
Also, could you explain why you start from `101`?

```suggestion
uint16 UNREADY=101
uint16 TIMEOUT=102
```